src/Mollie/WC/Payment/Order.php

```diff
@@ -331,12 +331,10 @@ public function onWebhookPaid( WC_Order $order, $payment, $paymentMethodTitle )
 				if ( version_compare( WC_VERSION, '3.0', '<' ) ) {
 					if ( Mollie_WC_Plugin::getDataHelper()->isSubscription( $order->id ) ) {
 						$this->deleteSubscriptionOrderFromPendingPaymentQueue( $order );
-						WC_Subscriptions_Manager::activate_subscriptions_for_order( $order );
 					}
 				} else {
 					if ( Mollie_WC_Plugin::getDataHelper()->isSubscription( $order->get_id() ) ) {
 						$this->deleteSubscriptionOrderFromPendingPaymentQueue( $order );
-						WC_Subscriptions_Manager::activate_subscriptions_for_order( $order );
```

Comment:
Probably, it makes sense to do refactoring here and in other places where similar code was changed in the same way. I'm writing this just because this code changed anyway and we have a chance to do it better without much effort.  For me, it looks like we are doing the same thing regardless of `WC` version check result:
```
$this->deleteSubscriptionOrderFromPendingPaymentQueue( $order );
```

The difference is only in the way of retrieving order id. That's why I'm suggesting something like 

```
$orderId = version_compare( WC_VERSION, '3.0', '<' ) ? $order->id : $order->get_id();

if ( Mollie_WC_Plugin::getDataHelper()->isSubscription( $orderId ) ) {
    $this->deleteSubscriptionOrderFromPendingPaymentQueue( $order );
}
```